### PR TITLE
chore(curriculum): validate colored markers project

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Contribute to #46540

### CODE REVIEW

**Review by DanielRosa74 — DISMISSED** [review]
LGTM

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61695197ac34f0407e339882.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ dashedName: step-1
 
 # --description--
 
-As you've seen in the previous projects, webpages should start with a `DOCTYPE html` declaration, followed by an `html` element.
+As you've seen in the previous projects, webpages should start with a `DOCTYPE html` declaration, followed by an `html` element with a `lang` attribute of `en`.
 
-Add a `DOCTYPE html` declaration at the top of the document, and an `html` element after that.
+Add a `DOCTYPE html` declaration at the top of the document, and an `html` element after that with `lang` attribute set to.
+
+`en`
```
Comment (moT01) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61695197ac34f0407e339882.md:14]:
```suggestion
Add a `DOCTYPE html` declaration at the top of the document, and an `html` element after that. Give the `html` element a `lang` attribute with `en` as its value.
```

**Review by moT01 — COMMENTED** [review]
Maybe outside the scope of the PR, but I made a couple minor grammar suggestions.

**Review by moT01 — APPROVED** [review]
LGTM 🎉 After tests pass

**Review by Sembauke — APPROVED** [review]

**Pull request merged**